src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -109,12 +123,62 @@ def create_job(email,
   return _batch_client().create_job(create_request)
 
 
-def main():
-  """Main function only used in development."""
-  email = 'untrusted-worker@clusterfuzz.google.com.iam.gserviceaccount.com'
-  # email = 'untrusted-worker@clusterfuzz-external.iam.gserviceaccount.com'
-  create_job(email=email)
-
-
-if __name__ == '__main__':
-  main()
+def get_spec(full_module_name, job):
+  """Gets the specifications for a job."""
+  platform = job.platform
+  command = utask_utils.get_command_from_module(full_module_name)
+  if command != 'fuzz':
+    platform += '-HIGH-END'
```

Comment:
It's a little unclear to me why these would be considered "high-end". Could you explain the reasoning here, seeing that this doesn't map to our previous definition of "high end"?